projects/canopy/src/lib/table/table.notes.ts

```diff
@@ -14,7 +14,35 @@ The table component provides a way to present tabular data in a responsive forma
 })
 ~~~
 
-and in your HTML:
+### Simple example
+
+~~~html
+<table lg-table>
+  <thead lg-table-head>
+    <tr lg-table-row>
+      <th lg-table-head-cell>Author</th>
+      <th lg-table-head-cell>Book</th>
+      <th lg-table-head-cell>Published</th>
+    </tr>
+  </thead>
+  <tbody lg-table-body>
+    <tr lg-table-row>
+      <td lg-table-cell>Orhan Pamuk</td>
+      <td lg-table-cell>Strangeness in my Mind</td>
+      <td lg-table-cell>2016</td>
+    </tr>
+    <tr lg-table-row>
+      <td lg-table-cell>Albert Camus</td>
+      <td lg-table-cell>The Plague</td>
+      <td lg-table-cell>1947</td>
+    </tr>
+  </tbody>
+</table>
+~~~
+
+### Table with expandable detail rows
+
+This table adds the \`\`LgTableRowToggleComponent\`\`, and then an expandable detail row. This creates a table that allows the user to expand a row for more information. This can be seen on the <a href="/story/components-table--detail">Expandable Detail story</a>.
```

Comment:
i would use a relative link rather than absolute just in case we ever hosted the app where the storybook wasn't at root

Reply:
Interestingly Storybook seems to make this URL relative anyway. It takes the URL fragment and adds it to `?path=`, resulting in `?path=/story/example`. If I try to make it relative, e.g. `story/example` or just `example`, it results in this following path: `?path=story/example` or `?path=example`, which breaks. So I think it has to be done like this?